src/components/Grid/Column.module.css

```diff
@@ -5,6 +5,12 @@
   box-sizing: border-box;
 }
 
+.columnheader {
+  text-transform: uppercase;
+  font-size: 11px;
```

Comment:
Should we use em instead of px ? any thoughts how will px behave in responsive webpage .

Reply:
I am also curious to learn this. I actually like fixed font size, but we might want to consider the case when the text is too long.

Reply:
Yes, this is a first iteration but we'll eventually need to add something like:

```css
white-space: nowrap;
overflow: hidden;
text-overflow: ellipsis;
```